### PR TITLE
[DOCS][ZEPPELIN-2140] Add docs for notebookRepo REST API

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Added a docs page for notebookRepo REST API based on [NotebookRepoRestApi.java](https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRepoRestApi.java). And this PR will cover "notebook repository reload endpoint" : #2043 as well. 

### What type of PR is it?
Documentation

### What is the Jira issue?
[ZEPPELIN-2140](https://issues.apache.org/jira/browse/ZEPPELIN-2140)

### How should this be tested?
Just checking screenshots will be faster! 

### Screenshots (if appropriate)
 - In navbar 
screen shot 2017-02-25 at 11 02 02 pm

 - API description list 
screen shot 2017-02-25 at 10 54 21 pm
screen shot 2017-02-25 at 10 54 29 pm
screen shot 2017-02-25 at 10 54 38 pm

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: docs/rest-api/rest-notebookRepo.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      <td>```http://[zeppelin-server]:[zeppelin-port]/api/notebook-repositories```</td>
+    </tr>
+    <tr>
+      <td>Success code</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <td>Fail code</td>
+      <td>
+        404 when the specified notebook repository doesn't existed <br/>
```
Comment (tae-jun) [inline docs/rest-api/rest-notebookRepo.md]: Isn't it a typo? `existed` 😄

**Comment by tae-jun** [comment]
LGTM!

File: docs/rest-api/rest-notebookRepo.md (hunk 1 above)
Comment (AhyoungRyu) [inline docs/rest-api/rest-notebookRepo.md]: @tae-jun Thanks for checking it out! Fixed in  [`c73e9b2`](https://github.com/apache/zeppelin/pull/2067/commits/c73e9b281569a8f16805929f7aef09b3b8549184)

**Comment by AhyoungRyu** [comment]
@tae-jun @felixcheung Thanks for approving! Will merge if there are no more comments on this.

**Pull request closed without merging**